command/import_test.go

```diff
@@ -274,691 +274,6 @@ func TestImport_providerConfigWithVarFile(t *testing.T) {
 	testStateOutput(t, statePath, testImportStr)
 }
 
-/*
```

Comment:
I must admit that I didn't look closely at them, but they all had names like `TestRefresh_...` which made me suspect that they had been copied from somewhere else as a basis.

Given that there's no comment here explaining what these are even here for, my vote would be to delete them. They'll be in the git history if we _do_ decide we want them, but without any comment explaining their purpose I don't see any way we'd reach such a conclusion.